apps/www/components/Sections/APISection.tsx

```diff
@@ -36,7 +36,7 @@ function APISection(props: Props) {
     <div className="grid grid-cols-12 lg:gap-16">
       <div className="col-span-12 pb-8 lg:col-span-5 xl:col-span-5">
         <h2 className="h3">{props.title}</h2>
-        <p className="p">{props.text}</p>
+        <div className="p">{props.text}</div>
```

Comment:
just to sanity check this was intentional?

Reply:
Yep. Sorry I should have left a comment on these seemingly random file changes. I was just fixes some hydration errors in www